Header header
JointRange[] joint
uint8 reqType
---
Header header
Param[] params
